src/main/java/com/iemr/flw/controller/BeneficiaryOTPGatewayController.java

```diff
@@ -0,0 +1,115 @@
+public class BeneficiaryOTPGatewayController {
+    final Logger logger = LoggerFactory.getLogger(this.getClass().getName());
+
+    @Autowired
+    private OTPHandler otpHandler;
+
+    @Operation(summary = "Send OTP")
+    @RequestMapping(value = "/sendOTP", method = RequestMethod.POST, produces = MediaType.APPLICATION_JSON,headers = "Authorization")
+    public String sendOTP(@RequestParam String phoneNumber) {
+        OutputResponse response = new OutputResponse();
+
+        try {
+
+            String success = otpHandler.sendOTP(phoneNumber);
+
+            if (success.contains("success"))
+                response.setResponse(success);
+            else
+                response.setError(5000, "failure");
+
+        } catch (Exception e) {
+            logger.error("error in sending OTP : " + e);
+            response.setError(5000, "error : " + e);
+        }
+        return response.toString();
+    }
+
+    @CrossOrigin()
+    @Operation(summary = "Validate OTP")
+    @RequestMapping(value = "/validateOTP", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON, produces = MediaType.APPLICATION_JSON,headers = "Authorization")
+    public String validateOTP(@Param(value = "{\"mobNo\":\"String\",\"otp\":\"Integer\"}") @RequestBody String requestOBJ) {
+
+        OutputResponse response = new OutputResponse();
+
+        try {
+            OTPRequestParsor obj = InputMapper.gson().fromJson(requestOBJ, OTPRequestParsor.class);
+
+            JSONObject responseOBJ = otpHandler.validateOTP(obj);
+            if (responseOBJ != null)
+                response.setResponse(responseOBJ.toString());
+            else
+                response.setError(5000, "failure");
+
+        } catch (Exception e) {
+            logger.error("error in validating OTP : " + e);
+            response.setError(5000, "error : " + e);
+        }
+        return response.toString();
+    }
+
+    @CrossOrigin()
+    @Operation(summary = "Save BeneficiaryId")
+    @RequestMapping(value = "/saveBeneficiaryId", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON, produces = MediaType.APPLICATION_JSON,headers = "Authorization")
+    public String saveBeneficiaryId(@RequestBody OtpRequestDTO requestOBJ) {
+
+        OutputResponse response = new OutputResponse();
+
+        try {
+
+            JSONObject responseOBJ = otpHandler.saveBenficiary(requestOBJ);
+            if (responseOBJ != null)
+                response.setResponse(responseOBJ.toString());
+            else
+                response.setError(5000, "failure");
+
+        } catch (Exception e) {
+            logger.error("error in validating OTP : " + e);
+            response.setError(5000, "error : " + e);
+        }
+        return response.toString();
+    }
+
+    @CrossOrigin()
+    @Operation(summary = "Resend OTP")
+    @RequestMapping(value = "/resendOTP", method = RequestMethod.POST, produces = MediaType.APPLICATION_JSON,headers = "Authorization")
```

Comment:
What's the difference between sendOTP and resendOTP API? Is this API endpoint required?